source/MaterialXGenArnold/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ add_library(MaterialXGenArnold STATIC
     ${materialx_header}
 )
 
+add_subdirectory(libraries)
```

Comment:
I'm curious as to why we don't just keep these in the libraries folder (like ogsfx) vs copying over. Assume we want to keep doing it one way or the other.

Reply:
It felt more modular and contained to keep everything Arnold related inside the same Arnold sub-directory. And copy the library data into its runtime place if Arnold is enabled. If we imagine support for other targets to be dropped in later, I think it makes sense to do it this way. But I'm open to other suggestions.

If we decide to handle it this way I can make the change to have the OgsFx target be handled in the same way.

Reply:
I like this way better. Let's cleanup OgsFx after. Thanks.